docs/redirects.md

```diff
@@ -0,0 +1,15 @@
+# Redirects
+
+If you need to set up a new redirect, you can do that in our notifications-broadcasts-infra repository.
+
+In there, head to `terraform/modules/govuk-alerts-website/files/redirects` and add your redirect to REDIRECTS constant, using the following format:
+
+```
+REDIRECTS = {"<uri of page you want to redirect from>": "<uri of page you want to redirect to>"}
+```
+
+Deploy-wise, you will need to do 3 deploys in quick succession:
+
+1. deploy a PR that adds the page to redirect to
+2. deploy the lambda change (thorugh terraform)
```

Comment:
minor typo
```suggestion
2. deploy the lambda change (through terraform)
```

also, do you need to know that it's through terraform? is it worth saying "through the govuk-alerts-infra deploy pipeline" or something?

Reply:
Ahh, I think I have been only remotely aware of govuk-alerts-infra concourse pipeline. So it deploys terraform for us? Where can I find this pipeline? I don't see it with our other pipelines?

Reply:
cool, thanks for the link, I will update the guidance 👍🏼